App.js

```diff
@@ -16,16 +17,16 @@ export default function App() {
           options={{headerShown: false}}
         />
         <Stack.Screen 
-          name="Dodge"
-          component={Dodge}
+          name="Battle"
+          component={Battle}
           options={{headerShown: false}}
         />
-         <Stack.Screen 
-          name="Results"
-          component={Results}
+        <Stack.Screen 
+          name="Dodge"
```

Comment:
I think you want "DodgeBallGame" to be created inside the "Battle" component.

I think it's slightly better organized that way, but more importantly you can create and re-create multiple different minigames within the Battle Screen.